### PR TITLE
remove Curricula, since since we're overhauling them

### PR DESCRIPTION
**Pull request by pfent**
Basically just removing the broken status quo, see #841.

I think in the future, we need to overhaul it properly. Most likely we can't reuse any of the current code, so I decided just to remove it all.

### CODE REVIEW

File: app/src/main/java/de/tum/in/tumcampusapp/component/other/generic/drawer/DrawerMenuHelper.kt [hunk 1]
```diff
@@ -93,8 +92,7 @@ class DrawerMenuHelper(private val mContext: Context, private val mDrawerLayout:
                 SideNavigationItem(R.string.person_search, R.drawable.ic_users, PersonSearchActivity::class.java, true, false),
                 SideNavigationItem(R.string.news, R.drawable.ic_rss, NewsActivity::class.java, false, false),
                 SideNavigationItem(R.string.barrier_free, R.drawable.ic_accessible, BarrierFreeInfoActivity::class.java, false, false),
-                SideNavigationItem(R.string.opening_hours, R.drawable.ic_time, OpeningHoursListActivity::class.java, false, false),
-                SideNavigationItem(R.string.study_plans, R.drawable.ic_study_plans, CurriculaActivity::class.java, false, false)
```
Comment (jacqueline8711) [inline app/src/main/java/de/tum/in/tumcampusapp/component/other/generic/drawer/DrawerMenuHelper.kt]: maybe remove 'study_plans' string from strings.xml for now?

**Comment by kordianbruck** [comment]
I'm all for removing the current status, once we have a go, that we have a different way of doing it. Right now we only scheduled a meeting for monday, that is not clear, what the result of it will be.

**Comment by pfent** [comment]
@kordianbruck since we met and we have the potential for for a proper API, could we fix the utterly broken status quo?

**Comment by kordianbruck** [comment]
@pfent I don't even know if we can get the API working. As you heard, the prospect of getting it is really slim.

**Comment by thellmund** [comment]
@pfent @kordianbruck What’s the status here?

**Pull request merged**